tests/api/common/test_mark_tasks.py

```diff
@@ -631,9 +632,10 @@ def test_set_success_dag_run_to_success(self):
         self._verify_task_instance_states(self.dag1, date, State.SUCCESS)
         self._verify_dag_run_dates(self.dag1, date, State.SUCCESS, middle_time)
 
-    def test_set_success_dag_run_to_failed(self):
+    @pytest.mark.parametrize("completed_state", [State.SUCCESS, State.FAILED])
+    def test_set_completed_dag_run_to_failed(self, completed_state):
```

Comment:
There were two identical test before:
`test_set_success_dag_run_to_failed ` and `test_set_failed_dag_run_to_failed `